README.adoc

```diff
@@ -174,14 +182,25 @@ query/SystemResource.java
 include::staging/query/src/main/java/io/openliberty/guides/query/SystemResource.java[]
 ----
 
-Replace the injection of the `system.user` and `system.password` properties with the [hotspot=userPassword file=3]`system.userPassword` property, and use the [hotspot=authHeader file=3]`systemUserPassword` variable to construct the authorization header.
+// file 4
+system/server.xml
+[source, XML, linenums, role='code_column']
+----
+include::finish/system/src/main/liberty/config/server.xml[]
+----
+
+Injects the [hotspot=userPassword file=3]`system.userPassword` property instead of the separate `system.user` and `system.password` properties. Then, use the [hotspot=authHeader file=3]`systemUserPassword` variable to construct the authorization header for the `system` service.
+
+Note that only registered users can access the system microservice. To authenticate a user, a [hotspot=basicRegistry file=4]`basicRegistry` element is configured in the `system` server configuration file that is at [hotspot file=4]`system/src/main/liberty/config/server.xml`. This `basicRegistry` element is a simple case for learning purposes, and for more information about different user registries, see the https://openliberty.io/docs/latest/user-registries-application-security.html[User registries documentation^].
+
+Because you are running the `query` service in dev mode, the changes that you made were automatically picked up. You're now ready to check out your application in your browser.
 
-Because you are running the `query` service in dev mode, the changes that you made were automatically picked up. Check out the service that you modified at the http://localhost:9085/query/systems/localhost URL.
+Point your browser to the http://localhost:9085/query/systems/localhost URL. You can see the current OS name and Java version in JSON format.
 
 // ===========================================================================================
-== Retrieving property as a list
+== Injecting property as a list
```

Comment:
what is the benefit of doing this- or an example case where it would be useful?

Reply:
Edited in https://github.com/OpenLiberty/draft-guide-microprofile-config-apis/pull/17/commits/324a3685c77af4f7dfe3051751832cbd805d968f